### ISSUE TITLE
High error rate on `dev` after db switch over

### ISSUE THREAD

**Issue by finestructure** [SwiftPackageIndex/SwiftPackageIndex-Server#1000]
```
[ CRITICAL ] ErrorPage.View 500: server: cannot cast type jsonb to integer (transformTypeCast)
```

Will roll back to docker db if it's related to the switch over.

**Comment by finestructure** [comment]
`dev` site seems to be fully operational though, so I'm not sure what's triggering it or what the actual effect is yet.

**Comment by finestructure** [comment]
😱  Hosted Azure dbs roll out with Postgres 9.6.

**Comment by finestructure** [comment]
Re-deploying with version 11 fixed this issue, but there seems to be a new, less frequent one:

```
[ ERROR ] Channel error caught. Closing connection. [psql_connection_id: BDB06496-7338-4FCC-BEFE-52E9813E3426, psql_error: PSQLError(base: PostgresNIO.PSQLError.Base.connectionError(underlying: NIOSSL.NIOSSLError.uncleanShutdown))]
```

<img width="989" alt="CleanShot 2021-03-24 at 10 03 07@2x" src="https://user-images.githubusercontent.com/65520/112283269-42daab80-8c88-11eb-89ac-a5ddf8c43973.png">

**Comment by finestructure** [comment]
Might be related to this: https://github.com/vapor/vapor/issues/2234 but I don't see how we would set `ignoreUncleanSSLShutdown` on the Postgres connection.

I'll keep this open for a bit to see if this is manageable or if we need to find ways to suppress this. Unless the level is critical, this shouldn't trigger rollbar items and overflow our account limit.

**Comment by finestructure** [comment]
Also: https://github.com/vapor/postgres-kit/issues/137, although that would indicate it's a fixed issue that's regressed.

**Comment by finestructure** [comment]
Fixed via #1041
